assets/javascripts/discourse/controllers/admin-plugins-discourse-subscriptions-products-show-plans-show.js.es6

```diff
@@ -47,6 +47,16 @@ export default Controller.extend({
   },
 
   actions: {
+    changeRecurring() {
```

Comment:
This could be shorter like:
```js
const recurring = this.get("model.plan.isRecurring");
this.set("model.plan.type", recurring ? RECURRING : ONE_TIME);
this.set("model.plan.isRecurring", recurring);
```